Documentation/docpage/rating.html

```diff
@@ -0,0 +1,287 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/Components/components.css" />
+    <link rel="stylesheet" href="/Documentation/docpage-css/docs.css">
+    <link rel="stylesheet" href="/Documentation/utilities/prismjs/prism.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+        integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous" />
+    <link rel="apple-touch-icon" sizes="180x180" href="/Documentation/docpage-assets/logo/favicon/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/Documentation/docpage-assets/logo/favicon/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/Documentation/docpage-assets/logo/favicon/favicon-16x16.png">
+    <link rel="manifest" href="/Documentation/docpage-assets/logo/favicon/site.webmanifest">
```

Comment:
Instead of repeating same css files you can make one file and import all the files inside it 
and import that file in html.